### DISCUSSION TITLE
How to connect to milvus cluster？

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15878] [Q&A and General discussion]
hi, I have deployed a cluster of milvus-server, I don't know how to connect to this cluster with connect API,
`connections.connect(host=host, port=port),`
which is the real "host" of milvus cluster below in the image?

![image](https://user-images.githubusercontent.com/74544828/156864678-989c5162-2eb3-4f92-9f7b-e1e05d882df6.png)

**Comment by XuanYang-cn** [comment]
It's milvus-proxy

**Reply by xiaofan-luan** [reply]
usually it is 19530, check hello milvus document

**Reply by engrabdu** [reply]
Hello @xiaofan-luan I want to deploy Milvus cluster but there is no proper documentation regarding milvus cluster ..like how to install mishard and start and how to communicate other nodes with cluster and the other thing after installation  and configuration how to check cluster status either its running or not, if you already deploy please share your experience or provide step by step installation and configuration files .i will be very thankful  to you for this .
Thanks again
